python/pyfunc-server/docker/base.Dockerfile

```diff
@@ -12,14 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM continuumio/miniconda3
+FROM condaforge/miniforge3
```

Comment:
What are the difference between miniconda with miniforge? Is there any risk of regression to our existing users?

Reply:
miniconda is company-driven (Anaconda) and miniforge is community-driven (mainly to resolve compatibility issues that miniconda has yet to provide). Also some licensing restrictions by miniconda according to thread below:

https://stackoverflow.com/questions/60532678/what-is-the-difference-between-miniconda-and-miniforge

Other than that, no real differences according to sources online.